src/components/Reports/PeopleReport/PeopleReport.jsx

```diff
@@ -451,16 +451,18 @@ class PeopleReport extends Component {
             </h4>
             <p>End Date</p>
           </div>
-          <div>
-            <h4>Bio {this.state.isBioPosted ? 'posted' : 'requested'}</h4>
-            {this.state.authRole == 'Administrator' || this.state.authRole == 'Owner' ? (
-              <ToggleSwitch
-                switchType="bio"
-                state={this.state.isBioPosted ? false : true}
-                handleUserProfile={onChangeBioPosted}
-              />
-            ) : null}
-          </div>
+          {!this.state.isBioPosted ? (
+            <div>
+              <h4>Bio {this.state.isBioPosted ? 'posted' : 'requested'}</h4>{' '}
+              {this.state.authRole == 'Administrator' || this.state.authRole == 'Owner' ? (
+                <ToggleSwitch
+                  switchType="bio"
+                  state={this.state.isBioPosted ? false : true}
```

Comment:
Since the condition _this.state.isBioPosted ? false : true_ is equivalent to _!this.state.isBioPosted_, it can be simplified to _!this.state.isBioPosted_.

It is generally a good practice to use strict equality (===) instead of loose equality (==) when comparing values. Therefore, updating the comparisons to === could also be used.

This is what gpt rewrote:

{!this.state.isBioPosted && (
  <div>
    <h4>Bio requested</h4>{' '}
    {this.state.authRole === 'Administrator' || this.state.authRole === 'Owner' ? (
      <ToggleSwitch
        switchType="bio"
        state={!this.state.isBioPosted}
        handleUserProfile={onChangeBioPosted}
      />
    ) : null}
  </div>
)}